apps/ledger-live-mobile/e2e/README.md

```diff
@@ -1,26 +1,187 @@
-# ledger-live-mobile e2e testing
```

Comment:
place this file under the wiki or in docs folder not `e2e` for consistency
rather create a `docs/E2E.md` file

Reply:
Ok 👍